config/schema/articles.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `bryancrowe_articles` (
```

Comment:
Maybe something to place under the tests domain?

Reply:
@pabloelcolombiano thanks for the review. I've moved the file.

Reply:
Thanks, looks good to me!